recipes/focus/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.3" %}
+{% set sha256 = "9f19df55c3f072bffd97cd148c1ad5768b230684f5bb26a44970a9c710b7951a" %}
+
+package:
+  name: focus
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/metageni/FOCUS/archive/{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: $PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - pytest
+  run:
+    - python >=3
+    - numpy >=1.12.1
+    - scipy >=0.19.0
+    - jellyfish
+
+test:
+  commands:
+    - focus --help
+
+about:
+  home: https://edwards.sdsu.edu/FOCUS
+  license: GPL-3.0
+  license_family: GPL
+  license_file: COPYING
+  summary: 'FOCUS is an innovative and agile model to profile and report organisms present in metagenomic samples based on composition usage without sequence length dependencies.'
+  dev_url: https://github.com/metageni/FOCUS
+
+extra:
+  skip-lints:
+    - should_be_noarch
```

Comment:
why did you skipped it and not used noarch?

Reply:
@bgruening  Fixed it. Let me know if it good to go now. Thanks

Reply:
@bgruening Fixed it. Let me know if it good to go now. Thanks